packages/mdc-slider/foundation.js

```diff
@@ -179,7 +187,10 @@ class MDCSliderFoundation extends MDCFoundation {
    * Called when the user's interaction with the slider ends
    */
   handleInteractionEnd() {
+    this.setActive_(false);
     this.adapter_.notifyChange();
+    this.adapter_.deactivateRipple();
+    this.adapter_.focusThumb();
```

Comment:
This is used to focus on the thumb when you dont click directly on the thumb. For example when you click on the slider track if this was not here it would not focus the thumb until you click on the thumb itself.